src/nvim/api/extmark.c

```diff
@@ -237,6 +281,16 @@ ArrayOf(Integer) nvim_buf_get_extmark_by_id(Buffer buffer, Integer ns_id,
           return rv;
         });
       }
+    } else if (strequal("verbose", k.data)) {
+      if (v->type == kObjectTypeBoolean) {
+        verbose = v->data.boolean;
+      } else if (v->type == kObjectTypeInteger) {
+        verbose = v->data.integer;
+      } else {
+        VALIDATE_EXP(false, "verbose", "Boolean or Integer", api_typename(v->type), {
+          return rv;
+        });
+      }
```

Comment:
can use `api_object_to_bool()` (it already includes the Integer fallback)

Reply:
Thanks, was looking for that function but couldn't find it